src/storage/storage.ts

```diff
@@ -57,10 +60,12 @@ export class Storage {
 
       process.env.STORAGE_EMULATOR_HOST = `http://${process.env.FIREBASE_STORAGE_EMULATOR_HOST}`;
     }
+    this.endpoint = (userEndpoint || process.env.STORAGE_EMULATOR_HOST || 'https://firebasestorage.googleapis.com') + '/v0';
```

Comment:
Since we already have one for the emulator host, does it make sense to use another env var for the endpoint override?